addons/calico/3.9.1/install.sh

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
How did you test this? When I create a new EC2 instance, sync the kurl repo and run `install.sh` with the aws addon enabled, `kube-controller-manager` is started with the right flags:

```kube-controller-manager --address=0.0.0.0 --allocate-node-cidrs=true --authentication-kubeconfig=/etc/kubernetes/controller-manager.conf --authorization-kubeconfig=/etc/kubernetes/controller-manager.conf --bind-address=127.0.0.1 --client-ca-file=/etc/kubernetes/pki/ca.crt --cloud-provider=aws --cluster-cidr=172.20.0.0/16 --cluster-signing-cert-file=/etc/kubernetes/pki/ca.crt --cluster-signing-key-file=/etc/kubernetes/pki/ca.key --configure-cloud-routes=true --controllers=*,bootstrapsigner,tokencleaner --kubeconfig=/etc/kubernetes/controller-manager.conf --leader-elect=true --node-cidr-mask-size=24 --requestheader-client-ca-file=/etc/kubernetes/pki/front-proxy-ca.crt --root-ca-file=/etc/kubernetes/pki/ca.crt --service-account-private-key-file=/etc/kubernetes/pki/sa.key --use-service-account-credentials=true```

`KubeProxyConfiguration` is also patched to use iptables instead of ipvs when the aws addon is installed.

Reply:
GCP single instance

Reply:
OIC. Kubeadm might enable node CIDR allocation when the aws cloud provider is used. Nodeless only works on AWS for now (so you need to enable nodeless + aws + calico).